lib/utils.js

```diff
@@ -28,9 +28,12 @@ module.exports = {
   },
 
   getField(parsedRecord, path) {
-    const value = _.get(parsedRecord.keys, path)
-    || _.get(parsedRecord.newDoc, path)
-    || _.get(parsedRecord.oldDoc, path);
+    const value = [parsedRecord.keys, parsedRecord.newDoc, parsedRecord.oldDoc]
+      .reduce((acc, entry) => {
+        return acc === undefined
+          ? _.get(entry, path)
+          : acc;
+      }, undefined);
```

Comment:
I think this might be first time in my life I have seen a reduce that does not need initial value??